pf-azure/pf-resource-az/resources/pf-log-analytics/functions/public/manager/new-pfloganalytics.ps1

```diff
@@ -0,0 +1,13 @@
+Function New-PfLogAnalytics
```

Comment:
maybe worth trying to have a generic New-PfDefinition that takes a type and does dynamic new to avoid having a New func for every def type?